### PR TITLE
Added Turkish

### PR DESCRIPTION
**Pull request by dkarakay**
I hope it is useful for your project. Happy HacktoberFest! #3

### CODE REVIEW

File: lang/tr.po [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+msgid "msg_list_badargument"
+msgstr "%s: --list seçeneği için delil 'hiç', 'ücretsiz', 'ücretli' or 'hepsi' olmalıdır"
+msgid "msg_unknown_option"
+msgstr "%s: bilinmeyen seçenek '%s'"
+msgid "msg_total_free"
+msgstr "%d ücretsiz paketler"
+msgid "msg_total_nonfree"
+msgstr "%d ücretli paketler"
+msgid "msg_rms_happy_ascii"
+msgstr "Yalnızca bedava paketler - rms gurur duyuyor!                           rms mutlu."
```
Comment (suve) [inline lang/tr.po]: This particular string should not exceed 79 characters. Please remove some spaces.

File: lang/tr.po [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+msgid "msg_rms_happy"
+msgstr "Yalnızca bedava paketler - rms gurur duyuyor!"
+msgid "msg_rms_disappoint_ascii"
+msgstr "10%% dan fazla paralı paket. Özgürlükten nefret mi ediyorsun??              rms kayboldu."
```
Comment (suve) [inline lang/tr.po]: Like above, these two strings should fit in 79 chars each.

File: man/tr.man [hunk 3]
```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
```
Comment (suve) [inline man/tr.man]: The section headers (`.SH`) should also be translated.

File: man/tr.man [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+\fBhttps://fedoraproject.org/wiki/Licensing:Main#Good_Licenses\fR
+The Fedora Project wiki sayfası kabul görmüş yazılım lisanları içerir.
+.TP
+\fBhttps://vrms.alioth.debian.org/\fR
+Orijinal programın ana sayfası \fBvrms\fR.
+.SH LICENCE
+Bu program GNU General Public lisansına göre ulaşılabilir ve Free Software
+Foundation tarafından yayınlanmıştır
+.SH AUTHOR
+Written in early 2017 by \fBsuve\fR.
```
Comment (suve) [inline man/tr.man]: This paragraph is not translated.

File: man/tr.man [hunk 5]
```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
+vrms-rpm - Ücretsiz olmayan yüklü yazılımların rapor edilmesi
+.SH SYNOPSIS
+\fBvrms-rpm\fR \fI[OPTION]\fR ...
```
Comment (suve) [inline man/tr.man]: The manpage uses `[seçenekler]` for options; I guess that should also be the case here.

**Comment by suve** [comment]
Sorry to bother you, but can I have an update on this?

**Comment by dkarakay** [comment]
I'll fix problems

**Comment by dkarakay** [comment]
@suve I think I fixed issues as your request above. You can check it.

**Comment by suve** [comment]
Looks good. Merged. Thanks!

**Pull request merged**